app/scripts/modules/amazon/src/serverGroup/details/rollback/rollbackServerGroup.html

```diff
@@ -10,13 +10,26 @@
         <div class="col-sm-3 sm-label-right">
           Restore to
         </div>
-        <div class="col-sm-6">
-          <ui-select ng-model="command.rollbackContext.restoreServerGroupName" class="form-control input-sm">
+        <div class="col-sm-8">
+          <ui-select ng-model="command.rollbackContext.restoreServerGroupName"
+                     class="form-control input-sm"
+                     ng-if="command.rollbackType === 'EXPLICIT'">
             <ui-select-match placeholder="Select...">{{ctrl.label($select.selected)}}</ui-select-match>
             <ui-select-choices group-by="ctrl.group" repeat="serverGroup.name as serverGroup in allServerGroups">
               <span ng-bind-html="ctrl.label(serverGroup)"></span>
             </ui-select-choices>
           </ui-select>
+          <div ng-if="command.rollbackType === 'PREVIOUS_IMAGE'" style="margin-top: 5px">
+            {{ previousServerGroup.name }}<br/>
+            <span class="small" ng-if="previousServerGroup.buildNumber">
+              <strong>Build</strong> #{{ previousServerGroup.buildNumber }} (no longer deployed)<br/>
```

Comment:
Should probably put `(no longer deployed)` next to the previous server group name. It's potentially misleading, as this build might still be deployed in a different cluster or region?

Reply:
That's fair.